app/vep/utils/cleanup.py

```diff
@@ -0,0 +1,53 @@
+import os
+import time
+import shutil
+
+class DiskCleanup:
+    def __init__(self):
+        self.name_spaces: list(str) = list(os.environ.get("NAMESPACE_LIST", "dev,prod,stage").split(","))
+        self.parent_directory = os.environ.get("UPLOAD_DIRECTORY")
+
+    def delete_old_temp_directories(self, days: int = 7):
+        threshold_time = time.time() - (days * 86400)
+
+        deleted_dirs = []
+        errors = []
+        for name_space in self.name_spaces:
+            vep_upload_directory = self.parent_directory + "/" + name_space
```

Comment:
The path of the parent dir should to be `<root>/<namespace>/vep` where `namespace` is one of `dev`,`internal`,`production`,`staging` (only `dev` and `production` currently in use).